hw/ip/otbn/doc/_index.md

```diff
@@ -313,14 +292,6 @@ The LOOP instruction allows for nested loops; the active loops are stored on the
 Each loop stack entry is a tuple of loop count, start address, and end address.
 The number of entries in the loop stack is implementation-dependent.
 
-### Call Stack
-
-A stack (LIFO) of function call return addresses (also known as "return address stack").
-The number of entries in this stack is implementation-dependent.
-
-The call stack is accessed through the `x1` GPR (return address).
-Writing to `x1` pushes to the call stack, reading from it pops an item.
```

Comment:
I'd prefer keeping this section. The call stack is a separate piece of state which needs potentially hardening and discussion. It's accessed through x1, but that's about all it has in common with the GPRs. Maybe push it up after the GPR section?

Reply:
Done, and I've avoided duplication by stripping stuff out of the GPRs section instead (which I think is probably cleaner anyway, thanks).